src/appengine/handlers/cron/triage.py

```diff
@@ -301,6 +411,12 @@ def get(self):
     # been removed.
     all_jobs = data_handler.get_all_job_type_names()
 
+    bug_filed_24_hours_per_job = {}
```

Comment:
We can  reduce a fair bit of complexity here by creating a Throttler class.

e.g.

```python

class Throtter:
  def __init__(self):
    self.bug_filing_max_24_hours_per_job = {}
    self.bug_filing_max_24_hours_per_project = {}
    self.bug_filed_24_hours_per_job = {}
    self.bug_filed_24_hours_per_project = {}

  def should_throttle(self, testcase):
    ...
```

Then we just create this Throttler object and pass it around instead of the caller having to create 4 dicts.